### PR TITLE
1699: Card design koblenz

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Adjust the koblenz card design

### Proposed changes

- replace logos
- set theme color
- adjust card branding
- add extra upscale for header logo

Note: I adjusted the header right logo a bit because fine adjustments with the current card branding are difficult. At some point we gonna have to create separate card template widgets per project

### Side effects

- none

### Testing

1. Start koblenz local and activate card
2. Check the other apps if the card design doesn't break

### Resolved issues

Fixes: #1699 
Fixes: #1741

### CODE REVIEW

File: frontend/build-configs/koblenz/index.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export const koblenzCommon: CommonBuildConfigType = {
     },
     categories: [9, 10, 11, 12, 13, 14, 15, 16, 17, 18],
     theme: {
-        primaryLight: "#8377A9",
-        primaryDark: "#8377A9",
+        primaryLight: "#d10074",
```
Comment (steffenkleinle) [inline frontend/build-configs/koblenz/index.ts:21]: :thought_balloon: I want a theme color :D Not related to your PR though

**Review by steffenkleinle — APPROVED** [review]
Tested on android, looks good

**Review by seluianova — COMMENTED** [review]
Is a pink background intended here? (I know we get it from the theme, but anyway)
image

**Comment by f1sh1918** [comment]
> Is a pink background intended here? image

The issue here is that material3 themes use surfaceTintColor. Usually a very light version of the primary color is used.
Since for koblenz the color is darker its still visible here.
The issue ist that if i override this color in general on lots of positions it will not be used like
image

So probably i have to override this color just for the dialogs

**Comment by seluianova** [comment]
> I will talk to toni in the afternoon

ready for review or you still want to clarify/update anything?

**Comment by seluianova** [comment]
just found another issue:
the card is not rendered correctly on the card verification.
image

```
======== Exception caught by rendering library =====================================================
The following assertion was thrown during layout:
A RenderFlex overflowed by 0.767 pixels on the bottom.

The relevant error-causing widget was: 
  Column Column:file:///home/tory/IdeaProjects/entitlementcard/frontend/lib/identification/id_card/card_content.dart:181:32
```

**Review by ztefanie — COMMENTED** [review]
Did just a quick review and did not test as two people already did. lgtm. Would have liked to have a background (image) on the card, but just personal preference.

**Comment by f1sh1918** [comment]
> Did just a quick review and did not test as two people already did. lgtm. Would have liked to have a background (image) on the card, but just personal preference.

Yes same for me but we don't have a proper design for that. This also leads to the issue that page background and card are both white and don't have a good contrast :(
Maybe we can improve that at a later point

**Comment by f1sh1918** [comment]
@seluianova i added a tint color to override this slight pink color for dialogs.
Fixed also the pixel overflow.
Additional design optimization like colors and contrasts will be done here
https://github.com/digitalfabrik/entitlementcard/issues/1742

**Pull request merged**